plugins/sep/sepsource.cc

```diff
@@ -294,9 +306,9 @@ int SepSource::TIFFReadScanline_(tiff *file, void *buf, uint32 row,
 }
 
 void SepSource::readCombinedScanline(std::vector<byte> &out, size_t line_nr) {
-  // There are 4 channels in out, so the number of bytes an individual
-  // channel has is one fourth of the output vector's size.
-  size_t size = out.size() / 4;
+  // There are n (=spp) channels in out, so the number of bytes an individual
+  // channel has is one nth of the output vector's size.
+  size_t size = out.size() / nr_channels;
```

Comment:
Shouldn't we get rid of the `white_ink` file in `SepSource`?

Reply:
Yes.
Removed